src/Model/Request/Transaction.php

```diff
@@ -18,14 +18,14 @@ class Transaction implements \JsonSerializable
     public $card = null;
 
     /**
-     * @param Card|Token $request
      * @param int $amount
      * @param string $currency
+     * @param Card|Token $request
      * @param bool $blocking
      * @param string $orderId
      * @throws Exception
      */
-    public function __construct( $request, $amount, $currency, $blocking = true, $orderId = null )
+    public function __construct( $amount, $currency, $request = null, $blocking = true, $orderId = null )
```

Comment:
Could roll back this change when the commit function call no longer requires a transaction object